app/models/miq_provision/options_helper.rb

```diff
@@ -56,7 +57,7 @@ def get_source
   end
 
   def get_hostname(dest_vm_name)
-    name_key = (source.platform == 'windows') ? :sysprep_computer_name : :linux_host_name
+    name_key = (source.try(:platform) == 'windows') ? :sysprep_computer_name : :linux_host_name
```

Comment:
Again, this may have unintended consequences if `source` doesn't respond to `platform`